src/components/admin/Tabs.tsx

```diff
@@ -9,7 +9,7 @@ function AdminTabs() {
     const intl = useIntl();
     const { pathname } = useLocation();
     const [selectedTab, setSelectedTab] = useState(0);
-    const { hasTenants } = useTenantDetails();
+    const hasAnyAccess = useUserInfoSummaryStore((state) => state.hasAnyAccess);
```

Comment:
This change will be all over. Anywhere we were checking `hasTenants` should now be `hasAnyAccess`. Please keep an eye and make sure if they are checking positive or negative and that was not lost with changes.